src/include/storage/data_table.h

```diff
@@ -217,9 +217,18 @@ class DataTable {
    */
   const BlockLayout &GetBlockLayout() const { return accessor_.GetBlockLayout(); }
 
+  /**
+   * @return reference to this DataTable's accessor
+   */
+  const TupleAccessStrategy &GetAccessor() const { return accessor_; }
```

Comment:
I think this should probably be `private` since it easily allows leaking implementation details.  @mbutrovich, do you have any thoughts on this?